x-pack/plugins/ml/public/settings/calendar/edit/directive.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import 'ngreact';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { uiModules } from 'ui/modules';
+const module = uiModules.get('apps/ml', ['react']);
+// TODO: change to relative paths
+import { checkFullLicense } from 'plugins/ml/license/check_license';
+import { checkGetJobsPrivilege } from 'plugins/ml/privilege/check_privilege';
+import { checkMlNodesAvailable } from 'plugins/ml/ml_nodes_check/check_ml_nodes';
+import { initPromise } from 'plugins/ml/util/promise';
+
+import uiRoutes from 'ui/routes';
+
+const template = `
+  <ml-nav-menu name="settings" />
+  <div class="mlCalendar_management">
+    <ml-new-calendar />
+  </div>
+`;
+
+uiRoutes
+  .when('/settings/calendars_list/new_calendar', {
+    template,
```

Comment:
You'll need to pull in the k7breadcrumbs from `x-pack/plugins/ml/public/settings/breadcrumbs.js` by rebasing to latest master

Reply:
👍 Doing this in a follow up PR to avoid backport messiness since the breadcrumbs update is only in master. This way I can backport this PR and just leave the route update follow-up PR in master. 😄